frontend/src/concepts/trustyai/context/useDoesTrustyAICRExist.ts

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { TrustyAIContext } from '~/concepts/trustyai/context/TrustyAIContext';
 import { SupportedArea, useIsAreaAvailable } from '~/concepts/areas';
+import { TrustyInstallState } from '~/concepts/trustyai/types';
 
 const useDoesTrustyAICRExist = (): boolean[] => {
   const trustyAIAreaAvailable = useIsAreaAvailable(SupportedArea.TRUSTY_AI).status;
-  const { hasCR } = React.useContext(TrustyAIContext);
+  const { statusState } = React.useContext(TrustyAIContext);
 
-  return [trustyAIAreaAvailable && hasCR];
+  return [trustyAIAreaAvailable && statusState.type === TrustyInstallState.INSTALLED];
```

Comment:
the CR could exists but be in a different state like ERROR or UNINSTALLING

Reply:
I get that -- but the use-case is to show the tab and the CR existing but in broken state does change functionality but I don't think it hurts. We could definitely update this to be not installed OR initial loading state. Less of an 
impact to the function that way I imagine.

Reply:
TODO for me